samples/packaged/CopilotChat/WebApp/src/App.tsx

```diff
@@ -53,19 +61,27 @@ const App: FC = () => {
                 <Login />
             </UnauthenticatedTemplate>
             <AuthenticatedTemplate>
-                <div style={{ display: 'flex', width: '100%', flexDirection: 'column', height: '100vh' }}>
-                    <div className={classes.header} >
-                        <Subtitle1 as="h1">Copilot Chat</Subtitle1>
-                        <Avatar
-                            className={classes.persona}
-                            key={account?.name}
-                            name={account?.name}
-                            size={28}
-                            badge={{ status: 'available' }}
-                        />
+                {appState === AppState.ProbeForFunction &&
+                    <FunctionProbe
+                        uri={import.meta.env.VITE_REACT_APP_FUNCTION_URI as string}
```

Comment:
can we remove Vite please? we need to simplify the toolchain

Reply:
We will discuss this during the live integration sync later today.